python/lsst/summit/utils/plotting.py

```diff
@@ -233,16 +220,21 @@ def plot(
                 norm = vis.ImageNormalize(imageData, interval=interval, stretch=vis.LinearStretch())
             case "sqrt":
                 norm = vis.ImageNormalize(imageData, interval=interval, stretch=vis.SqrtStretch())
+            case "pixInsight":
+                imageData = stretchDataPixInsight(imageData)
+                norm = None
```

Comment:
Since mypy complains about this line, you could probably do:
`norm = vis.ImageNormalize(imageData, stretch=vis.LinearStretch())`

Reply:
Oh nice, I actually didn't run a final mypy after that commit, thanks!